container/libcontainer/handler.go

```diff
@@ -132,6 +144,35 @@ func (h *Handler) GetStats() (*info.ContainerStats, error) {
 	return stats, nil
 }
 
+func processStatsFromProcs(rootFs string, cgroupPath string) (info.ProcessStats, error) {
+	var fdCount uint64
+	filePath := path.Join(cgroupPath, "cgroup.procs")
+	out, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return info.ProcessStats{}, fmt.Errorf("couldn't open cpu cgroup procs file %v : %v", filePath, err)
+	}
+
+	pids := strings.Split(string(out), "\n")
+	for _, pid := range pids {
+		dirPath := path.Join(rootFs, "/proc", pid, "fd")
+		fds, err := ioutil.ReadDir(dirPath)
+		if err != nil {
+			glog.V(4).Infof("error while listing directory %q to measure fd count: %v", dirPath, err)
+			continue
+		}
+		fdCount += uint64(len(fds))
+	}
+
+	processStats := info.ProcessStats{
+		// EOL is also treated as a new line while reading "cgroup.procs" file with ioutil.ReadFile.
```

Comment:
so does this mean one of the pids in our list above isn't actually a pid, but an empty string (or something else)?  Can we just remove this from the list of pids above instead of this?